resources/sql/queries.sql

```diff
@@ -98,6 +98,18 @@ VALUES (:resid, :organization, :owneruserid, :modifieruserid,
  /*~ (if (:endt params) */ :endt /*~*/ NULL /*~ ) ~*/
 );
 
+-- :name set-resource-state! :insert
+UPDATE resource
+SET
+  /*~ (when (boolean? (:enabled params)) */
```

Comment:
Should be no need to check `boolean?` here?

Reply:
It's a duplicate of `set-catalogue-item-state!`, which is in tests/testdata used to change just one of the fields.